<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CElevationLayerDialog</class>
 <widget class="QWidget" name="CElevationLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CElevationLayerDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_layer">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_layerName">
       <property name="text">
        <string>LayerName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_driveType">
       <property name="text">
        <string>DriveType</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_layerName"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_driveType">
         <item>
          <property name="text">
           <string>gdal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>flatgis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>bgc</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>feature_elevation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mbtiles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vpb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>xyz</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_file">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="radioBtn_singleData">
       <property name="text">
        <string>SingleData</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBtn_sliceData">
       <property name="text">
        <string>SliceData</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_path">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find_path">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="checkBox_cache">
       <property name="styleSheet">
        <string notr="true">color: rgb(249, 133, 0);</string>
       </property>
       <property name="text">
        <string>SetCache</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_cacheName">
       <property name="text">
        <string>CacheName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cachePath">
       <property name="text">
        <string>CachePath</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="lineEdit_cacheName"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="lineEdit_cachePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_find_cachePath">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
